test/e2e/framework.go

```diff
@@ -86,7 +86,7 @@ const (
 	busyboxImage        = "projects.registry.vmware.com/library/busybox"
 	nginxImage          = "projects.registry.vmware.com/antrea/nginx"
 	perftoolImage       = "projects.registry.vmware.com/antrea/perftool"
-	ipfixCollectorImage = "projects.registry.vmware.com/antrea/ipfix-collector:latest"
+	ipfixCollectorImage = "projects.registry.vmware.com/antrea/ipfix-collector:v0.3.1"
```

Comment:
Please remeber to update `ipfix-collector` version here as well next time.
https://github.com/vmware-tanzu/antrea/blob/master/ci/kind/test-e2e-kind.sh#L94

Reply:
Thanks for reminding. Will pay attention to that.